src/compressed_tensors/utils/helpers.py

```diff
@@ -119,3 +122,61 @@ def is_compressed_tensors_config(compression_config: Any) -> bool:
         return isinstance(compression_config, CompressedTensorsConfig)
     except ImportError:
         return False
+
+
+def getattr_chain(obj: Any, chain_str: str, *args, **kwargs) -> Any:
```

Comment:
do we always want to get the last `res = getattr(res, attr_name)`?
Will there be a case where we want the res that is not the last? 
If so, might be useful to yield res.

Reply:
Hm, I'm not really sure what you're indicating here. Can you give an example of what you're describing?